### PR TITLE
Bug 1856709: Remove single instance of whitespace from user input

### PR DESCRIPTION
**Pull request by bipuladh**
Screenshot from 2020-08-04 10-56-02

### CODE REVIEW

**Comment by bipuladh** [comment]
/hold
A better approach would be to do group capture

File: frontend/public/components/autocomplete.tsx [hunk 1]
```diff
@@ -57,8 +57,10 @@ const AutocompleteInput: React.FC<AutocompleteInputProps> = (props) => {
   React.useEffect(() => {
     if (textValue && visible && showSuggestions) {
       const processed = labelParser(data, labelPath);
+      // User input without whitespace
+      const processedText = textValue.trim().replace(/([^\s]*)\s*(=|^)\s*(.*)/, '$1$2$3');
```
Comment (kyoto) [inline frontend/public/components/autocomplete.tsx:61]: Couldn't we just do `replace(/\s*=\s*/, '=')`?

**Pull request merged**